### PR TITLE
Add Resources section to README.md

### PR DESCRIPTION
**Pull request by kohr-h**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,32 +26,29 @@ Features
...
+- [Installation guide](https://odlgroup.github.io/odl/getting_started/installing.html)
+- [Getting Started](https://odlgroup.github.io/odl/getting_started/getting_started.html)
+- [Code Examples](examples)
+- [API reference](https://odlgroup.github.io/odl/odl.html)
 
 License
--------
-GPL Version 3 or later. See [LICENSE](LICENSE) file.
-
-If you would like to get the code under a different license, please contact the developers.
```
Comment (adler-j) [inline README.md]: Sneaky

File: README.md [hunk 2]
```diff
@@ -26,32 +26,29 @@ Features
...
+    pip install odl
 
-This requires [pytest](http://pytest.org/latest/). See the [installation](https://odlgroup.github.io/odl/getting_started/installing.html) documentation for further information.
+The code is compatible with Python 2 and 3 through the `future` library. It is intended to work on all major platforms (GNU/Linux / Mac / Windows).
 
-ODL is compatible to Python 2 and 3 through the `future` library. It is intended to work on all major platforms (GNU/Linux / Mac / Windows).
+Resources
+=========
+- [ODL Documentation](https://odlgroup.github.io/odl/)
+- [Installation guide](https://odlgroup.github.io/odl/getting_started/installing.html)
```
Comment (adler-j) [inline README.md]: I would prefer if this link stayed with the Installing section, it flows better. I.e. keep the "See the [installation](https://odlgroup.github.io/odl/getting_started/installing.html) documentation for further information." line.

Comment (kohr-h) [inline README.md]: Well then, why not have both?

File: README.md (hunk 1 above)
Comment (kohr-h) [inline README.md]: ^^

File: README.md (hunk 2 above)
Comment (adler-j) [inline README.md]: Both is totally fine with me, i just dont want someone to try `pip install odl`, fail and then don't see the instructions on how to do more.

**Comment by kohr-h** [comment]
Comments fixed and rebased, merge after CI.

**Pull request merged**